templates/vue/src/components/Lightbox/media/MultiContentMedia/AccordionRows.vue

```diff
@@ -249,13 +249,15 @@ button[disabled] {
 }
 
 .button-row-trigger {
+  color: #111;
   background: none;
   width: 100%;
   text-align: left;
 }
 
 .accordion-row {
-  background: #262626;
+  color: #eee;
+  background: #eee;

```

Comment:
Having color and background the same means the text will be invisible. Is this a mistake?

Reply:
Yeah, it's a typo... it's supposed to be the background only

Reply:
yeah, it's supposed to be the background only... fixed.